### PR TITLE
Add "description" input parameter

### PR DESCRIPTION
**Pull request by kslr**

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@ echo "Setting up git machine..."
 git_setup
 
 echo "Forcing tag update..."
-git tag -a latest -m "This tag has been auto-generated by latest-tag" "${GITHUB_SHA}" -f
+git tag -a latest -m $INPUT_INTRODUCTION "${GITHUB_SHA}" -f
```
Comment (EndBug) [inline entrypoint.sh]: I think we need to put some quotes around it, don't we? Like: `"$INPUT_INTRODUCTION"`

File: action.yml [hunk 2]
```diff
@@ -7,4 +7,10 @@ runs:
 
 branding: 
   icon: tag
-  color: blue
\ No newline at end of file
+  color: blue
+
+inputs:
+  introduction:
```
Comment (EndBug) [inline action.yml]: `introduction` or `description`?

File: entrypoint.sh (hunk 1 above)
Comment (EndBug) [inline entrypoint.sh]: This is linked to one of the previous: do you want it to be named `description` or `introduction`?

**Comment by kslr** [comment]
I wrote the wrong README and should use "introduction".

**Comment by EndBug** [comment]
I've thought about that, and I think it'll be better to use the name "description" because that's how GitHub calls that part of a release. Don't worry, I'll take care of that ;)

**Comment by kslr** [comment]
Please use Squash

**Comment by EndBug** [comment]
@kslr Ok, everything is merged. Thank you a lot for your contribution 😄

**Pull request merged**